tests/mrna-drosopholia-DQ327736.fa

```diff
@@ -0,0 +1,7 @@
+>DQ327736 1
```

Comment:
better be real mRNA starting with `ATG`.

Reply:
could you kindly suggest a source of better test data?